### DISCUSSION TITLE
GitHub Copilot not working, Output log shows "CodeQuote disabled"

### DISCUSSION THREAD

**Discussion by picocodes** [community/community#50305] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I've tried the following:-

- [x] Reloading VSCode.
- [x] Restarted windows.
- [x] Reinstalled both VSCode and Copilot.
- [x] Resigned into my GitHub account in VS Code. 

No matter what I do, Copilot does not suggest anything. The output log shows the following:-

```bash
[INFO] [codequote] [2023-03-17T09:26:40.315Z] CodeQuote disabled
[ERROR] [promptlib proxy] [2023-03-17T09:26:41.127Z] {}
[ERROR] [promptlib proxy] [2023-03-17T09:26:41.130Z] {}
```

**Comment by Nitik5** [comment]
I'm facing the same issue, earlier it was working perfectly.
<img width="743" alt="Screenshot 2023-03-17 203928" src="https://user-images.githubusercontent.com/117816510/225944781-0746e092-9e46-419f-82b9-2f435777b49a.png">

**Reply by john-cockcroft** [reply]
Same for me, it is autocompleting comments but not giving any code hints

**Comment by picocodes** [comment, marked as answer]
I managed to fix this by rolling back Copilot to version v1.77.9225.

**Reply by shjyh** [reply]
it not works for me

**Reply by Helyk** [reply]
I was on Nightly and I don't see v1.77.9225 available but I rolled back to v1.77.9725 and that version works, despite showing the same message `CodeQuote disabled` upon loading.

**Reply by animate-object** [reply]
+1 9225 is up and running for me, some issue w/the latest build.

**Reply by Hexi1997** [reply]
I solved the problem by using https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly extension

**Reply by ramadhanafif** [reply]
How does you roll back to 1.77.9225?

**Reply by Hexi1997** [reply]
![image](https://user-images.githubusercontent.com/66669483/227121709-e98936cc-1da7-4b9b-9ddf-1e37836c7cfa.png)

**Reply by y4n6** [reply]
> ![image](https://user-images.githubusercontent.com/66669483/227121709-e98936cc-1da7-4b9b-9ddf-1e37836c7cfa.png)

This solved my problem, thank you!

**Reply by rajhlinux** [reply]
rolled back to v1.77.9725 doesn't work for me, same goes for v1.77.9700 (nightly copilot)

**Comment by rajhlinux** [comment]
Im getting the same error.

**Comment by s-sasaki-earthsea-wizard** [comment]
I was getting the same error, but reinstalling GitHub Copilot v1.77.9225 solved it.

**Comment by lipeng1667** [comment]
yeap, v1.77.9225  save my day

**Comment by mountxing** [comment]
yeap, update to v1.77.9225 solution

**Comment by XingnanJiang** [comment]
v1.77.9225 is good on M1
